services/clientSide/workspaceServices.ts

```diff
@@ -127,3 +127,24 @@ export const updateWorkspaceDetails = async (payload: { name: string }) => {
     workspaceStore.setLoading(false);
   }
 };
+
+/**
+ * @desc Make admin
+ * @param {*} payload
+ */
+export const makeAdmin = async (payload: { userId: string; role: string }) => {
```

Comment:
Define this interface in appTypes.tx and use everywhere we need

Reply:
Done